app/views/partials/foyer/individu-form.html

```diff
@@ -212,6 +212,16 @@
     <question>{{ isDemandeur() ? 'Êtes-vous' : 'Est-il/elle' }} étudiant boursier&nbsp;?</question>
   </yes-no-question>
 
+  <yes-no-question model="individu.aCharge"
+    ng-if="captureDemandeurAChargeParents()"
+    size="4"
+    >
+    <question>Êtes-vous à charge fiscale de vos parents&nbsp;?</question>
+    <help-block>
+      Répondez "Non" si vous avez déjà fait une déclaration de revenus en votre nom, ou si vous allez le faire pour l'année à venir.
```

Comment:
Itérons ;).

Techniquement, il n'est pas obligatoire de faire une déclaration de revenus si tu n'es pas imposable. En pratique, il est quand même recommandé de le faire car un avis de non imposition est une pièce justificative. Dans notre cas, je pense qu'il faut le faire (ou s'engager à le faire) pour pouvoir de demander une CMU quand on a moins de 25 ans.